src/modules/msg.ts

```diff
@@ -0,0 +1,23 @@
+export const okHand: string = ':ok_hand:';
```

Comment:
`okHand` is not really needed... Just makes for additional complexity and imports.

If this is used standalone in a message then you can edit the message to be a bit more verbose.